packages/integrations/cloudflare/test/basics.test.js

```diff
@@ -5,28 +5,29 @@ import * as cheerio from 'cheerio';
 describe.skip('Basic app', () => {
```

Comment:
Is the skip intended?

Reply:
Looks like it was skipped in https://github.com/withastro/astro/pull/4117. I found the bug though that had us skipped the test and will submit that separately.